src/pages/lib-v1/auth/emailpassword/q/platform/[platform].mdx

```diff
@@ -258,19 +237,31 @@ async function signIn() {
 </Block>
 </BlockSwitcher>
 
+The `signIn` API response will include a `nextStep` property, which can be used to determine if further action is required. It may return the following next steps:
+
+- `CONFIRM_SIGN_IN_WITH_NEW_PASSWORD_REQUIRED` - The user's password must be reset before the sign in can be completed.
+- `CONFIRM_SIGN_IN_WITH_CUSTOM_CHALLENGE` - The sign-in must be confirmed with a custom challenge response.
+- `CONFIRM_SIGN_IN_WITH_TOTP_CODE` - The sign-in must be confirmed with a TOTP code from the user.
+- `CONTINUE_SIGN_IN_WITH_TOTP_SETUP` - The TOTP setup process must be continued.
+- `CONFIRM_SIGN_IN_WITH_SMS_CODE` - The sign-in must be confirmed with a SMS code from the user.
+- `CONTINUE_SIGN_IN_WITH_MFA_SELECTION` - The user must select their mode of MFA verification before signing in.
+- `DONE` - The sign in process has been completed.
+
```

Comment:
we are missing the `RESET_PASSWORD` and `CONFIRM_SIGN_UP` steps. The  `RESET_PASSWORD` step is returned when an admin forces to reset the user's password and `CONFIRM_SIGN_UP` step is returned when the end user forgot to confirm their account during `signUp`